microsoftgraph/microsoft-graph-explorer-v4
# Fix: responsive views

## Overview

Changes the mobile breakpoint to where the screen breaks, It then gives it a mobile appearance so that icons do not overlap

Breaks down the app component so that it is easy to read and edit

Fixes #485 


### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* Resize and zoom the screen
* Notice the change in layour
* Start after checking out this branch
* Include any setup required, such as bundling scripts, restarting services, etc.
* Include test case, and expected output
